packages/quantic/README.md

```diff
@@ -50,6 +50,22 @@ Start the server.
 - You can also run this command from the command line. `sfdx force:lightning:lwc:start`
 - View the server at http://localhost:3333/.
 
+## Run Cypress for Quantic components
+
+Ref: [https://docs.cypress.io/](https://docs.cypress.io/)
+
+- All the tests will need to be under folder `cypress/integration`
+
+
+First, start the local development server by running:
+
+       npm run start
```

Comment:
```suggestion
       `npm run start`
```

Reply:
The command already appear as a code block (because of the leading whitespaces). However, I'll update with backticks to make it more readable and have syntax highlighting.

    ```bash
    npm run start
    ```